### ISSUE TITLE
1.21 version requires tommylib

### ISSUE THREAD

**Issue by dayofpi** [thomasglasser/Sherds-API#7]
When trying to run datagen, I get this error message:

```net.fabricmc.loader.impl.FormattedException: Some of your mods are incompatible with the game or each other!
A potential solution has been determined, this may resolve your problem:
	 - Install tommylib, version 15.0.0 or later.
More details:
	 - Mod 'Sherds API' (sherdsapi) 5.2.6 requires version 15.0.0 or later of tommylib, which is missing!
```

But there are no instructions on the wiki for how to install tommylib

**Comment by thomasglasser** [comment]
The wiki has been updated. Let me know if it still doesn't work after including the TommyLib repo

**Comment by dayofpi** [comment]
I get the following error when trying to load gradle changes now.

```
> Could not resolve all files for configuration ':modImplementation'.
   > Could not resolve dev.thomasglasser.sherdsapi:sherdsapi-fabric-1.21:5.2.6.
     Required by:
         project :
      > Module 'dev.thomasglasser.sherdsapi:sherdsapi-fabric-1.21' has been rejected:
           Cannot select module with conflict on capability 'dev.thomasglasser.sherdsapi:sherdsapi:5.2.6' also provided by [dev.thomasglasser.sherdsapi:sherdsapi-common-1.21:5.2.6(runtimeElements)]
   > Could not resolve dev.thomasglasser.sherdsapi:sherdsapi-common-1.21:5.2.6.
     Required by:
         project : > dev.thomasglasser.sherdsapi:sherdsapi-fabric-1.21:5.2.6
      > Module 'dev.thomasglasser.sherdsapi:sherdsapi-common-1.21' has been rejected:
           Cannot select module with conflict on capability 'dev.thomasglasser.sherdsapi:sherdsapi:5.2.6' also provided by [dev.thomasglasser.sherdsapi:sherdsapi-fabric-1.21:5.2.6(runtimeElements)]
```

**Comment by thomasglasser** [comment]
Should be fixed as of 5.2.8
